client/tests/e2e/specs/test_folders_page.ts

```diff
@@ -206,9 +206,6 @@ describe('Check folders page', () => {
     cy.get('.folder-container').find('.file-list-item').eq(1).find('ion-checkbox').invoke('show').click();
     cy.get('#button-moveto').contains('Move to').click();
     cy.get('.folder-selection-modal').find('.ms-modal-header__title').contains('Move one item');
-    cy.get('.folder-selection-modal')
```

Comment:
This code should be replaced by a code that check that the current path is correct. Same for every deletion in this file.